conf.py

```diff
@@ -139,7 +139,7 @@
 
 # If not '', a 'Last updated on:' timestamp is inserted at every page bottom,
 # using the given strftime format.
-#html_last_updated_fmt = '%b %d, %Y'
+html_last_updated_fmt = '%b %d, %Y'
```

Comment:
What do you think about using ISO format?

Reply:
I like the format `'%b %d, %Y'`, it is the same one we have in docs.python.org 😃